engine/sdk/src/test/wscript

```diff
@@ -102,20 +102,45 @@ def build(bld):
 
     # Create the test files and try to build them
     with open('src/test/test_include.cpp', 'rt', encoding='utf-8') as f:
-        template = f.read()
+        template_cpp = f.read()
 
-    for index, include in enumerate(includes):
-        basename = os.path.basename(include).replace('.', '_') + '_%d.cpp' % index
-        source = 'test_include_%s' % basename
+    with open('src/test/test_include.c', 'rt', encoding='utf-8') as f:
+        template_c = f.read()
 
-        source_node = bld.path.find_or_declare(source)
+    # Make sure that our C api holds up using only C compiler
+    c_includes = ['extension.h',
+                  'configfile.h',
+                  'resource.h']
```

Comment:
We test build the C api headers using C compiler, to make sure it's strict C code.